.github/workflows/ci.yml

```diff
@@ -39,6 +39,11 @@ jobs:
         with:
           version: 4.0.13
 
+      - name: Setup Rust for WebAssembly
+        run: |
+          rustup target add wasm32-unknown-unknown
+          cargo install -f wasm-bindgen-cli --version 0.2.100
```

Comment:
Instead of hardcoding the version here, can we use cargo to install dependencies?

Reply:
I tried putting it in Cargo.toml at first but that didn't work, it's not possible to run the tool that way.